### PR TITLE
[17.0][IMP] web_timeline: Update vis-timeline-graph2d lib: 7.7.0 -> 7.7.3

### PR DESCRIPTION
**Pull request by carlos-lopez-tecnativa**
no breakchanges, see https://github.com/visjs/vis-timeline/releases/tag/v7.7.3
TT50999
@Tecnativa @pedrobaeza @CarlosRoca13 @chienandalu  could you please review this

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]

**Review by chienandalu — COMMENTED** [review]
isn't the diff much larger than needed? I don't see that many changes in the related release notes :thinking: for example, looks like css stuff is just moved from one place to another in the css file...

File: web_timeline/static/lib/vis-timeline/vis-timeline-graph2d.css [hunk 1]
```diff
@@ -889,6 +889,21 @@
   border-style: solid;
   border-radius: 4px;
 }
+.vis .overlay {
```
Comment (pedrobaeza) [inline web_timeline/static/lib/vis-timeline/vis-timeline-graph2d.css:892]: Why increasing the diff moving the styles ?

**Comment by pedrobaeza** [comment]
Yes, and the library JS seems also very different. Maybe one version is minified and the other not?

**Comment by carlos-lopez-tecnativa** [comment]
I only copied the files generated by executing the `npm run build` command. The result is similar to previous updates, as seen in the commit.

https://github.com/OCA/web/pull/2378/commits/94323daa0cb913e5862c7f7cbb92a63c24db095d

**Comment by pedrobaeza** [comment]
Well, it's the double in fact (48k additions instead of 24k). Not sure if there can be any command line for changing somehow the result.

**Comment by carlos-lopez-tecnativa** [comment]
> Well, it's the double in fact. Not sure if there can be any command line for changing somehow the result.

I'm not sure, but if @chienandalu or @CarlosRoca13 can help me with the command used in the previous case, that would be great. The difference is mainly in the indentation.

image

**Comment by chienandalu** [comment]
> The difference is mainly in the indentation.

That's not hard to deal with. Just select the whole block and indent it to the right :)

**Comment by carlos-lopez-tecnativa** [comment]
> > The difference is mainly in the indentation.
> 
> That's not hard to deal with. Just select the whole block and indent it to the right :)

Yes, but I meant that the current spacing is 2
image

and now it's 4. I think the new one is better, although it shows more differences. In the next updates, the differences should be smaller.
image

**Comment by chienandalu** [comment]
> Yes, but I meant that the current spacing is 2

Ah, ok. In that case I think is ok :) I just saw your first capture where all the code was pulled to the left

**Comment by pedrobaeza** [comment]
Can't you configure a different indentation or open it and change it for minimizing the diff?

**Comment by carlos-lopez-tecnativa** [comment]
> Can't you configure a different indentation or open it and change it for minimizing the diff?

I took the file directly from https://unpkg.com/vis-timeline@7.7.3/standalone/umd/vis-timeline-graph2d.js
If I manually adapt the code, the result will be very different again in the next update.

**Comment by pedrobaeza** [comment]
Are we sure the next time it will be indented the same way?

**Review by pedrobaeza — APPROVED** [review]
As the diff seems to not be avoidable, let's merge it:

/ocabot merge patch

**Pull request merged**